OJP/OJP_FareSupport.xsd

```diff
@@ -446,11 +460,18 @@
 					<xs:documentation>URL for booking.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="BookingNote" type="InternationalTextStructure" minOccurs="0">
-				<xs:annotation>
-					<xs:documentation>Note about booking the FLEXIBLE LINE.</xs:documentation>
-				</xs:annotation>
-			</xs:element>
+			<xs:choice>
+				<xs:element name="BookingNote" type="BookingNoteType" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>Note about booking the LINE (deprecated).</xs:documentation>
```

Comment:
If we are doing all kind of breaking the API that hard, why this deprecation here?
The BookingNote was introduced with 4e90159b "Aligning BookingArrangements more closely with NeTEx", which is not part of a published version. I opt to remove the choice and BookingNote completely in favor of only a BookingNotes element

Reply:
adapted